demo/tests/test_perfs_rec.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+import sys
+import os
+import json
+
+from django.utils.version import get_complete_version
+from django.test import TestCase as OrigTestCase, Client
+
+try:
+    from django.urls import reverse
+except ImportError:
+    from django.core.urlresolvers import reverse
+
+from django_perf_rec import TestCaseMixin
+
+
+class TestCase(TestCaseMixin, OrigTestCase):
+    """
+    Add use perf rec mixin
+    """
+
+    c = Client()
```

Comment:
You don't need to instantiate `Client` here, it's already accessible via `self.client` in a [Django TestCase](https://docs.djangoproject.com/en/1.8/topics/testing/tools/#default-test-client).